sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -120,22 +129,30 @@
         PROPERTY_AZURE_TRACING_DISABLED,
     };
 
-    /**
+    /*
+     * The base set of configurations that is loaded at the start of the application. This will be used to pre-populate
+     * all further Configurations created.
+     */
+    private static final Configuration BASE_CONFIGURATIONS = loadBaseConfiguration();
+
+    /*
      * Gets the global configuration shared by all client libraries.
      */
-    private static final Configuration GLOBAL_CONFIGURATION = new Configuration();
+    private static final Configuration GLOBAL_CONFIGURATION = BASE_CONFIGURATIONS.clone();
```

Comment:
Do we need both BASE_CONFIGURATIONS and GLOBAL_CONFIGURATION? We can directly use the global configurations as the base when creating a new instance. Both of these should be immutable anyway.

Reply:
Removed